staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -791,6 +791,12 @@ const (
 
 	// SEVLabel marks the node as capable of running workloads with SEV
 	SEVLabel string = "kubevirt.io/sev"
+
+	// Flavor is the name of a VirtualMachineFlavor
+	Flavor string = "kubevirt.io/flavor"
+
+	// ClusterFlavor is the name of a VirtualMachineClusterFlavor
+	ClusterFlavor string = "kubevirt.io/cluster-flavor"
```

Comment:
Two things.

1. can you append `Annotation` to the end of these variable names. So, `ClusterFlavor` would become `ClusterFlavorAnnotation`. The reason being, if i saw `v1.Flavor` referenced in the code, I'd think that was a reference to the flavor struct, and not a string referencing the annotation to retrieve a flavor's name. 
2.  can you append `name` to the end of the values. so `kubevirt.io/flavor` would become `kubevirt.io/flavor-name`